dev-libs/xapian/xapian-1.4.25.recipe

```diff
@@ -109,6 +111,8 @@ INSTALL()
 	prepareInstalledDevelLib libxapian
 	fixPkgconfig
 
+	ln -sf $libDir/libxapian.so.$libVersion $libDir/libxapian.so
```

Comment:
Maybe this could be done with a sed command, but haven't figured that out, libxapian.so is referenced in $libDir/cmake/xapian/xapian-config.cmake with:
```
SET(XAPIAN_SHARED_LIBRARY "/packages/xapian-1.4.25-2/.self/lib/libxapian.so" CACHE FILEPATH "Shared Xapian library")
```

Reply:
The cmake file should be patched to use `$developLibDir` instead. This is used to find the library to link to.

Reply:
That doesn't do the trick, as it will mix up $libDir and $developLibDir for finding libraries/headers (tried to my best knowlidge, hence I added the comment here). :) IIRC it mixed up header path versus library path when checking up on khelpcenter (ps, this wasn't/isn't an issue for the other ports using this).